src/rest/Ratings/addRatingToPlace.ts

```diff
@@ -120,23 +133,33 @@ export const addRatingToPlace = async (
 	}
 
 	// request body starts as strings, convert to float if present
-	const brailleAsNum = braille ? parseFloat(braille) : null;
-	const fontReadabilityAsNum = fontReadability ? parseFloat(fontReadability) : null;
-	const staffHelpfulnessAsNum = staffHelpfulness ? parseFloat(staffHelpfulness) : null;
-	const navigabilityAsNum = navigability ? parseFloat(navigability) : null;
-	const guideDogFriendlyAsNum = guideDogFriendly ? parseFloat(guideDogFriendly) : null;
+	const guideDogFriendlyAsNum: number | null = guideDogFriendly
+		? parseFloat(guideDogFriendly)
+		: null;
+	const isMenuAccessibleAsNum: 0 | 1 | null = isMenuAccessible
```

Comment:
can someone make `0 | 1 | null` a reusable type instead of repeating it so much?